portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,29 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random quote to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomQuote() {
+  const quotes =
+      ["Hello world!", "That's rough buddy", "She didn't say you did this, she said Hugh did this", "The grass is not always greener on the other side, it's greener where you water it", "The krusty krab pizza is the pizza for you and me", "What the foop? It's time to go girl"];
```

Comment:
Looks good, but I recommend wrapping lines so that each line stays under 80 characters. You'll find that a lot of people do this to make it easier to read a file without scrolling side to side, especially when youve got several open at once!

Reply:
Noted. Thanks